src/web/app/components/error-report/error-report.component.ts

```diff
@@ -20,13 +21,20 @@ export class ErrorReportComponent implements OnInit {
   content: string = '';
   requestId: string = '';
   sendButtonEnabled: boolean = true;
+  errorReportEnabled: boolean = true;
   errorReportSubmitted: boolean = false;
+  csrfErrorMessages: string[] = ['Missing CSRF token.', 'Invalid CSRF token.'];
+  readonly supportEmail: string = environment.supportEmail;
 
   constructor(private errorReportService: ErrorReportService,
               private ngbActiveModal: NgbActiveModal,
               private statusMessageService: StatusMessageService) {}
 
   ngOnInit(): void {
+    if (this.csrfErrorMessages.includes(this.errorMessage)) {
+      this.sendButtonEnabled = false;
```

Comment:
Feels a bit redundant since the button wouldn't even be loaded yes?

Reply:
True, can remove this line